test/cbmc/proofs/DNS/DNSgetHostByName/DNSgetHostByName_harness.c

```diff
@@ -160,6 +207,21 @@ void harness()
 {
     size_t len;
 
+    pxNetworkEndPoints = ( NetworkEndPoint_t * ) safeMalloc( sizeof( NetworkEndPoint_t ) );
```

Comment:
Here again we are using safeMalloc. Why not use malloc only?

Reply:
fixed